### PR TITLE
Remove `threeDSecureVerification` property

### PR DESCRIPTION
**Pull request by sestevens**
### Summary of changes

 - Remove `threeDSecureVerification` from `BTDropInRequest`; instead of setting this, merchants should set `threeDSecureRequest` in order to request 3DS

 ### Checklist

 - [x] Added a changelog entry

- @sestevens

### CODE REVIEW

**Review by sshropshire — APPROVED** [review]
🌊

File: Sources/BraintreeDropIn/BTDropInController.m [hunk 1]
```diff
@@ -268,9 +268,6 @@ - (void)threeDSecureVerification:(BTPaymentMethodNonce *)tokenizedCard {
     paymentFlowDriver.viewControllerPresentingDelegate = self;
 
     BTThreeDSecureRequest *request = self.dropInRequest.threeDSecureRequest;
-    if (!request) {
-        request = [[BTThreeDSecureRequest alloc] init];
-    }
```
Comment (scannillo) [inline Sources/BraintreeDropIn/BTDropInController.m:273]: I was thinking - this PR should also remove wherever the logic was that set the amount on the `ThreeDSecureRequest` from the `BTDropInResult` ... however after searching the SDK I don't see this happening anywhere. So this looks sufficient, then!

**Review by scannillo — APPROVED** [review]
YEE HAW

File: Sources/BraintreeDropIn/BTDropInController.m (hunk 1 above)
Comment (sestevens) [reply Sources/BraintreeDropIn/BTDropInController.m:273]: Yeah, I was looking for that too! I think we must have removed `amount` previously.

**Pull request merged**